skyportal/facility_apis/winter.py

```diff
@@ -89,6 +91,28 @@ def _build_payload(self, request):
 
         return target
 
+    def schedule_name(request):
+        content = request.transactions[0].response["content"]
+        content = json.loads(content)
+        # the WINTERAPI POST response has a "msg" key,
+        # that contains Schedule name is <schedule_name> if successful
+        if "msg" in content:
+            # just to that we are not sensible to the exact format of the message
```

Comment:
sensible is a weird word.

Reply:
@mcoughlin that's the french in me sorryyyyy. Will fix ASAP. I also fixed the import already